packages/remix-dev/compiler.ts

```diff
@@ -353,7 +361,8 @@ async function createBrowserBuild(
     browserRouteModulesPlugin(config, /\?browser$/),
     emptyModulesPlugin(config, /\.server(\.[jt]sx?)?$/),
     NodeModulesPolyfillPlugin(),
-  ];
+    await getYarnPnpPlugin(),
+  ].filter((p): p is esbuild.Plugin => !!p);
```

Comment:
Let's make it more obvious that we're returning a `boolean` here

```suggestion
  ].filter((p): p is esbuild.Plugin => Boolean(p));
```